proposals/proposal_1/lib/src/items/item_list.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/material.dart';
+import 'package:proposal_1/src/items/item.dart';
+import 'package:proposal_1/src/items/item_controller.dart';
+import 'package:proposal_1/src/items/item_details.dart';
+
+class ItemList extends StatefulWidget {
+  final ItemController controller;
+
+  const ItemList({Key key, @required this.controller}) : super(key: key);
+
+  @override
+  _ItemListState createState() => _ItemListState();
+}
+
+class _ItemListState extends State<ItemList> {
+  Item _selectedItem;
+
+  void _onItemTapped(Item item) {
+    setState(() {
+      _selectedItem = item;
+    });
+  }
+
+  @override
+  void initState() {
+    // Begin loading the users when the Item List is first shown.
+    widget.controller.initialize();
+
+    super.initState();
```

Comment:
minor but shouldn't `super.initState()` generally be called first when overriding `initState`?